ui/cypress/plugins/index.js

```diff
@@ -27,5 +150,78 @@ module.exports = (on, config) => {
     );
   }
 
+  on('task', {
+
+    'debug': (...args) => {debug(...args); return true;},
```

Comment:
This is to get access to this really nice node-console debug on tests and on when running on github!

Use like:

```
cy.task('debug', myVar);
```

Where `myVar` can be any param allowed to `console.log`.

Locally, this will be printed on the terminal log for cypress, like:

![Screenshot from 2023-08-18 11-48-04](https://github.com/jataware/dojo/assets/1967061/36b637f3-da85-4765-92e9-c6303e3c24a0)

and on github action jobs:

![Screenshot from 2023-08-18 11-48-52](https://github.com/jataware/dojo/assets/1967061/12d2469a-fe4e-4c89-8cf6-a3122579dcd8)